src/core/reducers/addons.js

```diff
@@ -172,6 +172,8 @@ export function createInternalAddon(
         updateURL: apiAddon.theme_data.updateURL,
         version: apiAddon.theme_data.version,
       }),
+      // We should be using this property.
+      themeData: removeUndefinedProps({ ...apiAddon.theme_data }),
```

Comment:
I think we need to introduce a mapper and call this like `themeData: createInternalThemeData(apiAddon)` because of several reasons:

* Even though it adds verbosity, I think it's helpful to see all the fields assigned one by one
* When a new field is added to the API, we won't introduce the new field by surprise
* We already have to fix the mapping for one of the fields (`description`) because of a bug
* We may need to re-map others later on too.

You can move [all of this stuff](https://github.com/willdurand/addons-frontend/blob/11d457c73768a0e6b4861ef8803e3bc17ba485b1/src/core/reducers/addons.js#L146-L174) into  a new `createInternalThemeData()` function.